contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -214,6 +215,27 @@ static table_variable_satisfies_vacuum_hook_type prev_table_variable_satisfies_v
 static table_variable_satisfies_vacuum_horizon_hook_type prev_table_variable_satisfies_vacuum_horizon = NULL;
 static drop_relation_refcnt_hook_type prev_drop_relation_refcnt_hook = NULL;
 
+
+/* To cache oid of sys.varchar */
+static Oid sys_varcharoid = InvalidOid;
+
+static Oid get_sys_varcharoid()
```

Comment:
This is good, Can we declare this as extern and make use in 3 other places in extension?

Reply:
Done